SpeechResponder/eddi.fr.json

```diff
@@ -1365,7 +1365,7 @@
ardCommodityAmount} tonne{if event.rewardCommodityAmount > 1:s} de {event.rewardCommodity}\r\n        {OneOf(\"confirmé en chargement\",\"confirmé en soute\",\"reçu\",\"embarqué en récompense\")}.\r\n    }\r\n|elif event.rewardMaterial && event.rewardMaterial != \"\":\r\n    {if event.reward > 0:\r\n        {OneOf(\"En plus, \",\"Et, \")}\r\n    }\r\n    {event.rewardMaterialAmount} unité{if event.rewardMaterialAmount > 1:s} de {event.rewardMaterial}\r\n    {OneOf(\"ont été reçu\",\"donné en récompense\")}.\r\n}\r\n\r\n{if ranktype = \"RANKFED\" || ranktype = \"RankFed\":\r\n    {Pause(500)}\r\n    {set rank to cmdr.federationrating.rank + 1}\r\n\r\n    Congratulations {OneOf(\"Commandeur {cmdr.name}\",\"Sieur\")}! Vous\r\n        {OneOf(\"{OneOf('élevez votre position','êtes promu')}\",\r\n            \"montez en grade\")}\r\n        dans la marine fédérale\r\n\r\n    {if rank = 3:\r\n        , et pouvez désormais avoir accés au Fédéral Dropship\r\n    |elif rank = 4:\r\n        , et vous pouvez accéder au système sol {P(\"Sol\")}\r\n    |elif rank = 5:\r\n        , et pouvez désormais avoir accés au Fédéral Assault Ship.\r\n        Et, vous recevez les permis d'accés pour les systèmes {P(\"Vega\")} et {P(\"Beta Hydri\")}\r\n    |elif rank = 6:\r\n        , et, vous recevez le permis d'accés pour le système PLX 695\r\n    |elif rank = 7:\r\n        , et pouvez désormais avoir accés au Fédéral Gunship.\r\n        Et vous recevez le permis d'entrer dans le système Ross 128\r\n    |elif rank = 8:\r\n        , et vous recevez le permis d'entrer dans le système  {P(\"Exbeur\")}\r\n    |elif rank = 10:\r\n        , et vous recevez le permis d'entrer dans le système  {P(\"Hors\")}\r\n    |elif rank = 12:\r\n        , et pouvez désormais avoir accés à la Corvette Fédérale\r\n    |elif rank = 14:\r\n        , et vous avez atteint le plus haut rang dans la fédération\r\n    }.\r\n\r\n|elif ranktype = \"RANKEMP\" || ranktype = \"RankEmp\":\r\n    {Pause(500)}\r\n    {set rank to cmdr.empirerating.rank + 1}\r\n\r\n    Congratulations {OneOf(\"Commandeur {cmdr.name}\",\"Sieur\")}! Vous\r\n        {OneOf(\"{OneOf('élevez votre position','êtes promu')}\",\r\n            \"montez en grade\")}\r\n    dans l'Empire.\r\n\r\n    {if rank = 1:\r\n        Vous pouvez désormais avoir accés à l'Impérial Eagle\r\n    |elif rank = 3:\r\n        Vous pouvez désormais avoir accés à l'Imperial Courier\r\n    |elif rank = 4:\r\n        Vous recevez l'honneur d'un accés libre pour le système {P(\"Archenar\")}\r\n    |elif rank = 7:\r\n        Vous pouvez désormais avoir accés à l'Imperial Clipper.\r\n        Et vous recevez le permis d'entrer dans le système {P(\"Summer-land\")}\r\n    |elif rank = 10:\r\n        Et vous recevez le permis d'entrer dans le système {P(\"Facece\")}\r\n    |elif rank = 12:\r\n        Vous pouvez désormais avoir accés à l'Imperial Cutter\r\n    |elif rank = 14:\r\n        et vous avez atteint le plus haut rang dans l'Empire\r\n   }.\r\n}\r\n\r\n",
```

Comment:
As addressed in #2021, consider replacing `{mission.name}` with `{mission.localisedname}` when describing a completed community goal mission. 

There are several other changes to the English in that PR which might be worth considering for the French translation. The English text is available for reference from the PR.

Reply:
Small changes were also made to English scripts in #2022 and #2023 that might be worth considering for the French translation.